xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/editors/dataeditors.css

```diff
@@ -323,20 +323,24 @@ div#xwikiobjects input[type="radio"], div#xwikiclassproperties input[type="radio
   text-decoration: underline;
   background-color: $theme.highlightColor;
 }
-
+#add_xobject .buttonwrapper input {  
+  vertical-align: top;
+}
+#add_xproperty .buttonwrapper input {
+  vertical-align: top;
+  margin: 0px;
+}
```

Comment:
When I said to put the CSS selectors on separate lines I didn't meant to duplicate the styles. I simply wanted you to format the CSS code so that it's easier to read. And again, the unit is not important when the value is ``0`` so it should be omitted.

Reply:
Oh! Got it. Thanks